.github/CODEOWNERS

```diff
@@ -257,6 +257,7 @@
 /core/domain/question*.py @aks681 @vinitamurthi
 /core/storage/question/ @aks681 @vinitamurthi
 /core/templates/dev/head/components/concept-card/ @sophiewu6 @vinitamurthi
+/core/templates/dev/head/components/skill-mastery/ @sophiewu6 @vinitamurthi
```

Comment:
Should we add this in sorted order? (This will make it easier to maintain.)

Reply:
Done.